docs/data/routes/docs/nextjs/getting-started/walkthrough-jsscreate/en.md

```diff
@@ -3,4 +3,67 @@ name: walkthrough-jsscreate
 routeTemplate: ./data/component-templates/article.yml
 title: Walkthrough with `jss create`
 ---
-# Walkthrough: with `jss create`
\ No newline at end of file
+# Walkthrough: Using the JSS CLI to get started with JSS and Next.js 
+
+You can create a JSS Next.js application by using the JSS command-line tool (CLI). 
+
+> When using this approach, you will need to perform additional steps to connect to a Sitecore instance.
+
+We recommend this approach for Sitecore developers that use the [code-first developer workflow](https://jss.sitecore.com/docs/fundamentals/dev-workflows/code-first). 
```

Comment:
"or who wish to evaluate the JSS developer experience without installing Sitecore" ?